src/LibCURL.jl

```diff
@@ -21,27 +14,67 @@ function __init__()
     global cacert = MozillaCACerts_jll.cacert
 end
 
-include("lC_exports_h.jl")
-include("lC_common_h.jl")
-include("lC_curl_h.jl")
+const IS_LIBC_MUSL = occursin("musl", Base.BUILD_TRIPLET)
```

Comment:
Can `Sys.MACHINE` be used here instead of `Base.BUILD_TRIPLET`? The latter seems to be Julia 1.6 specific.

Reply:
I thought about this but I failed to find the definition of `Sys.MACHINE` for `musl` in the Julia source code.

Reply:
`BUILD_MACHINE` seems to be generated [here](https://github.com/JuliaLang/julia/blob/4211c3f6c83678cbed968c61755476ccc491eb6e/Make.inc#L769) and `BUILD_TRIPLET` [here](https://github.com/JuliaLang/julia/blob/4211c3f6c83678cbed968c61755476ccc491eb6e/Make.inc#L1163) which are transferred into julia code [here](https://github.com/JuliaLang/julia/blob/4211c3f6c83678cbed968c61755476ccc491eb6e/base/Makefile#L45) and [here](https://github.com/JuliaLang/julia/blob/4211c3f6c83678cbed968c61755476ccc491eb6e/base/Makefile#L88). Do these not suffice for `musl`?

Checking with a `julia:1.6.1-alpine3.13` docker container, I get `Sys.MACHINE` as `"x86_64-linux-musl"`

Reply:
Thanks for the info! Then I think it's ok to replace `Base.BUILD_TRIPLET` with `Sys.MACHINE`.